osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerRoomManager.cs

```diff
@@ -53,20 +52,18 @@ private void joinMultiplayerRoom(Room room, string password, Action<Room> onSucc
         {
             Debug.Assert(room.RoomID.Value != null);
 
-            multiplayerClient.JoinRoom(room, password).ContinueWith(t =>
+            multiplayerClient.JoinRoom(room, password).FireAndForget(() =>
             {
-                if (t.IsCompletedSuccessfully)
-                    Schedule(() => onSuccess?.Invoke(room));
-                else if (t.IsFaulted)
-                {
-                    const string message = "Failed to join multiplayer room.";
+                Schedule(() => onSuccess?.Invoke(room));
+            }, exception =>
+            {
+                const string message = "Failed to join multiplayer room.";
 
-                    if (t.Exception != null)
-                        Logger.Error(t.Exception, message);
+                if (exception != null)
+                    Logger.Error(exception, message);
```

Comment:
The `IsFaulted` branch of `FireAndForget()` already does error logging:

https://github.com/ppy/osu/blob/f71f761a3f22005b107fbbf494578871670cb76d/osu.Game/Online/Multiplayer/MultiplayerClientExtensions.cs#L19-L36

so wouldn't this mean that a possible error would get logged twice (with differing messages and severity, to boot) here?